### DISCUSSION TITLE
How can I enable the chat?

### DISCUSSION THREAD

**Discussion by kovacsj100** [owncast/owncast#3026] [Q&A] [answered]
Hello Everyone,

Please, help me, the chat doesn't work on my owncast page.
How can I enable it?
I suspect, I have websocket problem, because the error message in the browser console the next:
p.js:635 Owncast service worker registration failed! TypeError: Cannot read properties of undefined (reading 'register')
 
Thank you very much, in advance!
Best regards,
Jozsef Kovacs

**Comment by gabek** [comment, marked as answer]
The service worker isn't related to chat, but maybe all Javascript has failed for you for one reason or another. Does chat work on any browser or OS for you?

**Reply by kovacsj100** [reply]
Thank you very much, Gabek, on Firefox it is working.

**Reply by Widow1975taz** [reply]
Burnie Huntley stole my accouts

**Comment by Surajsuman047** [comment]
Everything work from your heart beat and also mind thinking patarn so many time you what but not to do same so first do and think after
